controllers/topics.controllers.js

```diff
@@ -0,0 +1,15 @@
+const { selectTopics } = require('../models/topics.models');
+
+exports.getTopics = (req, res, next) => {
+    selectTopics()
+    .then((topics) => {
+        if(topics.length === 0) {
+            res.status(404).send({msg: 'not found'});
+        } else {
+            res.status(200).send({ topics });
+        }
```

Comment:
This is not the ideal way to handle errors. We should be assuming that the .then is the successful path and only needs to include the successful response. This should not be where we handle errors.